tests/integration/test_memory_management.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from autogpt.chat import save_memory_trimmed_from_context_window
+from autogpt.config import Config
+from autogpt.memory import get_memory
+from tests.utils import requires_api_key
+
+
+@pytest.fixture
```

Comment:
There's a couple of cleaner ways to do this.

First would be to use an outer `'''` -- this would avoid having to escape `\"` everywhere

Second would be to actually write it as a Python dict:
```
test = {
    'thoughts' = {
        : 
```
and then `return [{'content': json.dumps(test, indent=4), ...`